crypto/opcode.py

```diff
@@ -480,17 +486,27 @@ def __init__(self, value, name, length, opfunc):
 opcodeArray[OP_IF]                  = opcode(OP_IF, "OP_IF", 1, opcodeIf)
 opcodeArray[OP_NOTIF]               = opcode(OP_NOTIF, "OP_NOTIF", 1, opcodeNotIf)
 opcodeArray[OP_VERIF]               = opcode(OP_VERIF, "OP_VERIF", 1, opcodeReserved)
-opcodeArray[OP_VERNOTIF]            = opcode(OP_VERNOTIF, "OP_VERNOTIF", 1, opcodeReserved)
+opcodeArray[OP_VERNOTIF]            = opcode(
+    OP_VERNOTIF, "OP_VERNOTIF", 1, opcodeReserved
+)
```

Comment:
How bad was Black's attempt? Would this look better if we just did three lines for every entry, regardless of length?

Reply:
> How bad was Black's attempt?

It would lose the equal signs alignment, while wrapping too long lines as it is here.

> Would this look better if we just did three lines for every entry, regardless of length?

Probably not, and Black wouldn't do it anyway, it would still have to be done manually.